backend/tracim_backend/lib/core/user.py

```diff
@@ -873,8 +874,12 @@ def create_minimal_user(
         save_now=False,
     ) -> User:
         """Previous create_user method"""
-        if not email and not username:
-            raise EmailOrUsernameRequired("Email or username is required to create an user")
+        if not email:
+            if not username:
+                raise EmailOrUsernameRequired("Email or username is required to create an user")
+            if self._config.EMAIL__REQUIRED:
+                raise EmailRequired("Email is required to create an user")
+
```

Comment:
```suggestion
        if not email:
            if self._config.EMAIL__REQUIRED:
                raise EmailRequired("Email is required to create an user")

            if not username:
                raise EmailOrUsernameRequired("Email or username is required to create an user")

```

It would generate a more useful error message for the case `EMAIL__REQUIRED` I think.